public/scripts/extensions/tts/alltalk.js

```diff
@@ -388,7 +388,7 @@ class AllTalkTtsProvider {
     }
 
     async fetchRvcVoiceObjects() {
-        if (this.settings.server_version !== 'v2') {
+        if (this.settings.server_version !== 'v1') {
```

Comment:
So it looks like the comment below is slightly wrong here now? Skipping is done only for IS NOT v1
I think it would be better to write it as a positive check, rather than negative, i.e.
```suggestion
        if (this.settings.server_version === 'v2') {
```

Reply:
Happy to do it either way. Have made the change as suggested.

Thanks